src/page-generator.ts

```diff
@@ -189,13 +191,19 @@ export async function generatePages(
     throw new Error(`Pages folder ${pagesFolder} does not exist`);
   }
 
-  const preferredRunner = RUNNERS.find(
-    (runner) => runner.command === wizardState.preferredRunnerForNextra
-  )!;
+  const preferredRunner = RUNNERS.find((runner) => {
+    return runner.command === wizardState.preferredRunnerForNextra;
+  });
+
+  if (!preferredRunner) {
+    throw new Error(
+      `Preferred runner for \`nextra\` not found: ${wizardState.preferredRunnerForNextra}`
+    );
+  }
```

Comment:
logic change 1

Reply:
seems good there's already a check in app.ts so this shouldn't fire anyway